src/components/tag/tag.js

```diff
@@ -198,8 +198,8 @@ const Tag = props => {
                   background: inherit;
                   border-style: solid;
                   border-width: 1px 1px 1px 1px;
-                  &:hover,
-                  &:focus {
+                  :not(:disabled)&:hover,
+                  :not(:disabled)&:focus {
```

Comment:
How dare you use css for this!

Reply:
If it can be done with CSS, I will do it with CSS 😎